content/articles/getting-started-with-unit-testing-with-spring-boot/index.md

```diff
@@ -31,31 +28,31 @@ It's good to simulate database-interacting beans and disable Spring Boot test DB
 Whenever an expert developer teaches unit testing to beginners, he should always ensure that the tests are correct and straightforward. To achieve this, he should keep the test with low cyclomatic complexity. Cyclomatic complexity is a coding statistic that shows how many different execution pathways a procedure can take. Developers are less likely to introduce problems when working on code with a lesser complexity since it is easier to comprehend and maintain.
 ### Reasons for testing...
 #### Repositories
-The repository is the place where all the data is stored. It is tested to ensure that specs or relationships have been correctly implemented.
+The repository is a bridge between the application and the database. It is tested to ensure that relationships between database and the applications have been correctly implemented.
 #### Services
 This is the layer where all the business logic is implemented. It is tested to make sure that the business logic is correct.
 ### How to test controllers
-Let us now look at how to test controllers in Spring Boot. To accomplish this, you will need to import some dependencies using the `Spring Initializer` and `IntelliJ IDEA`. `IntelliJ IDEA` lets us create a Spring Boot application using the Spring Initializr service. Open `file>new>project` and select `Spring Initializr` as shown below:
+Let us now look at how to test controllers in Spring Boot. To accomplish this, you will need to import some dependencies using the `Spring Initializer` and `IntelliJ IDEA`. In, `IntelliJ IDEA` lets us create a Spring Boot application using the Spring Initializr service. Open `file>new>project` and select `Spring Initializr` as shown below:
```

Comment:
I would add a disclaimer here that to do this you need IntelliJ IDEA ultimate, otherwise, you'll need to go to https://start.spring.io